spring-cloud-stream-binder-rabbit-docs/src/main/asciidoc/overview.adoc

```diff
@@ -113,6 +125,34 @@ delayedExchange::
   The `x-delayed-type` argument is set to the `exchangeType`.
 +
 Default: `false`.
+dlqDeadLetterExchange::
+  if a DLQ is declared, a DLX to assign to that queue
```

Comment:
This is a common pattern - lots of questions on SO about this.

`someExchange -> mainqueue -> reject -> DLQ`

configure the DLQ with a TTL (say 30000).
configure the DLQ with a DLX = "someExchange"

Rejected/expired messages get a cumulative `x-death` header and you can determine how many times around the loop we have traveled.

I will add a section to the docs to describe the pattern.